packages/website/src/components/MentorBenefits.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import { BenefitCard, BenefitCardProps } from '@coderscamp/ui/components/BenefitCard';
+import { Center } from '@coderscamp/ui/components/Center';
+import { Grid } from '@coderscamp/ui/components/Grid';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import {
+  BookOpenReaderIcon,
+  BrainIcon,
+  ChampagneGlassesIcon,
+  EyeIcon,
+  FileSparklesIcon,
+  FireIcon,
+  GraduationCapIcon,
+  HandshakeSimpleIcon,
+  SeedlingIcon,
+  SitemapIcon,
+  UsersIcon,
+  UserTieIcon,
+} from '@coderscamp/ui/icons';
+
+export interface Benefit extends BenefitCardProps {
+  id: string;
```

Comment:
💭 All benefits have unique titles so I think there is no need to add an ID to them.